.ci/openshift-ci/build_install.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This script is evoked within an OpenShift Build to product the binary image,
+# which will contain the Kata Containers installation into a give destination
+# directory.
+#
+
+set -e
+
+export PATH="${GOPATH}/bin:$PATH"
+
+cidir="$(dirname "$0")/.."
+source /etc/os-release
+
+source "${cidir}/lib.sh"
+
+[ -z "$1" ] && die "Usage: $0 path/to/install/dir"
+export DESTDIR="$1"
+info "Build and install Kata Containers at ${DESTDIR}"
+
+# Let the scripts know it is in OpenShift CI context.
+export OPENSHIFT_CI="true"
+
+# This script is evoked within a variant of CentOS container image in the
+# OpenShift Build process. So it was implemented for running in CentOS.
+[ "$ID" != "centos" ] && die "Expect the build root to be CentOS"
+# The scripts rely on sudo which is not installed in the build environment.
+yum install -y sudo
```

Comment:
This presupposes the script is being run as `root` or `yum` is setuid. Might be worth adding an `id -u` check?

Reply:
All the scripts on this PR are executed by `root` user. Just to make it clear I will add the check as you suggested.